server/jetstream_cluster_1_test.go

```diff
@@ -8924,6 +8924,28 @@ func TestJetStreamClusterDontReviveRemovedStream(t *testing.T) {
 	require_Error(t, err, ErrJetStreamStreamNotFound)
 }
 
+func TestJetStreamClusterCreateR3StreamWithOfflineNodes(t *testing.T) {
```

Comment:
Let's extend this test to bring the two servers back and make sure that `FOO` comes up as expected.

Reply:
Done, now bringing the offline servers back, and ensuring all is synced up. Especially for the offline server that got the stream assigned and needs to catchup.